doc/userdoc/installation/cmake_options.rst

```diff
@@ -120,6 +120,8 @@ Generic build configuration
 | ``-Dwith-intel-compiler-flags=[OFF|<list;of;flags>]``| User defined flags for the Intel compiler                        |
 |                                                      | [default='-fp-model strict']. Separate multiple flags by ';'.    |
 +------------------------------------------------------+------------------------------------------------------------------+
+| ``-Dwith-cpp-std=[OFF|<C++ standard>]``              | C++ standard to use for compilation [default='c++11'].           |
```

Comment:
Does `-Dwith-cpp-std=OFF` actually work and result in the default value?

Reply:
It does, `OFF` would just mean the default value. But because it's a bit confusing I have corrected the CMake variable now.